### ISSUE TITLE
Oh-my-zsh two-line PS1 not displaying correctly

### ISSUE THREAD

**Issue by shayani** [warpdotdev/Warp#1152]
### Discord username (optional)

shayani#9033

### Describe the bug

```text
I'm using the BIRA theme from Oh-my-zsh, that is a two-line theme. The PS1 is not being displayed correctly, even with the "Honor PS1" setting on.
```

### To Reproduce

Steps to reproduce:

1. Install Oh-my-zsh
2. Change theme to "bira"

### Expected behaviour

```text
Expected to see the Bira theme the way it should be, in two lines.
```

### BIRA theme in WARP:

![Captura de Tela 2022-04-21 às 12 23 34](https://user-images.githubusercontent.com/1488409/164491924-99a26070-e975-4d2b-9313-d1c0aa8154e9.png)

### BIRA theme in iTerm2, the way it sould appear:
![Captura de Tela 2022-04-21 às 12 25 17](https://user-images.githubusercontent.com/1488409/164492338-d1419103-df51-4200-9646-a176347ccf40.png)

### Operating System

MacOS

### OS Version

v0.2022.04.18.09.08.stable_03

**Comment by guisfits** [comment]
Same problem with "starship" zsh-theme

**Warp**
<img width="1032" alt="Screen Shot 2022-05-05 at 19 57 18" src="https://user-images.githubusercontent.com/20538361/167040862-ee32edb9-2a12-4261-a165-cae2d32ea890.png">

**Tabby**
<img width="1032" alt="Screen Shot 2022-05-05 at 19 57 58" src="https://user-images.githubusercontent.com/20538361/167040884-fdc95fe1-1a97-458b-a1af-fee8d66e6e70.png">

**Comment by JunyuMu** [comment]
Same problem with "spaceship" zsh-theme

**Warp**

<img width="355" alt="WX20220508-010904@2x" src="https://user-images.githubusercontent.com/15103817/167264676-2b68015e-3b34-4ea6-9ac7-140c205b3fc4.png">


**iTerm2**

<img width="388" alt="WX20220508-010810@2x" src="https://user-images.githubusercontent.com/15103817/167264694-0208ba6f-fede-4ed7-becc-d97990db9232.png">

**Comment by JunyuMu** [comment]
Find a solution: exec `zsh`

<img width="375" alt="WX20220508-020217@2x" src="https://user-images.githubusercontent.com/15103817/167266386-0495c493-030d-46f9-ba9d-72885bc35eec.png">

**Comment by tcvieira** [comment]
Have similar problem and with the @JunyuMu "solution" my cursor keeps scrolling up after clear (ctrl + l) and all commands stay in the same warp block because of it.

**Comment by elviskahoro** [comment]
Gonna close this in favor of:
#1015

tldr:
We don't support multi-line prompts yet (not sure whether this would be on the roadmap). The Input Editor is a completely separate UI element from the terminal grid, this is what enables support for a modern text editing experience.
